alpha/action/diff.go

```diff
@@ -148,11 +159,33 @@ func LoadDiffIncludeConfig(r io.Reader) (c DiffIncludeConfig, err error) {
 			errs = append(errs, fmt.Errorf("package at index %v requires a name", pkgI))
 			continue
 		}
+		if pkg.Range != "" && (len(pkg.Versions) != 0 || len(pkg.Bundles) != 0) {
+			errs = append(errs, fmt.Errorf("package at index %v has an invalid settings for range/versions/bundles", pkgI))
```

Comment:
Could we suggest a correction here? Or point to docs or something? Something like "Please only use Range OR Versions and Bundles, or some such...

Or maybe: `package at index %v contains range %s, %d versions and %d bundles. range and versions and/or bundles are mutually exclusive. Please only specify one (or please see docs at ...)`. 

It would just be nice to know the remedy.

Reply:
I did add the comment on this exclusivity on the struct (see above) but I will improve the message on the error a bit for clarity.